scripts/src/pullrequest/prartifact.py

```diff
@@ -25,9 +27,11 @@ def get_modified_charts(api_url):
 
 def save_metadata(directory, vendor_label, chart, number):
     with open(os.path.join(directory, "vendor"), "w") as fd:
+        print(f"add {directory}/vendor as {vendor_label}")
         fd.write(vendor_label)
 
     with open(os.path.join(directory, "chart"), "w") as fd:
+        print(f"add {directory}/chart as {chart}")
         fd.write(chart)
```

Comment:
Are those `print` statements left overs?

Reply:
Yes, but I left them in because they only come out when there is a failure and can be useful. We have way too few of these in the flows.